### PR TITLE
feat:added results page

### PR DESCRIPTION
**Pull request by aishwarya-bm**
added landing page & results page

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
all code looks clean and good

File: stylesheets/result.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.option-correct {
+    background-color: var(--success-txt);
+    border: 0;
```
Comment (srejitk) [inline stylesheets/result.css:3]: Any particular reason for giving border:0?

**Review by srejitk — APPROVED** [review]
Looks good. Really liked the code structure and application of utils. Just left a small suggestion.

File: screens/lastquestion.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            <div class="question-current">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Quidem aspernatur laudantium excepturi velit maiores quia non rem placeat ad esse?</div>
+
+            <ul class="list-no-bullet question-options-container children-stacked-center text-center">
+                <li class="list-item option-item">Option A</li>
+                <li class="list-item option-item">Option B</li>
+                <li class="list-item option-item">Option C</li>
+                <li class="list-item option-item">Option D</li>
+            </ul>
+
```
Comment (tanishq20) [inline screens/lastquestion.html:52]: Avoid giving extra spaces.

File: stylesheets/result.css (hunk 1 above)
Comment (aishwarya-bm) [reply stylesheets/result.css:3]: The options have border style, but when I show the correct & wrongs answers i want them as filled colors with green & red, so kept the border 0

**Review by tanishq20 — APPROVED** [review]
Code looks very clean

File: screens/lastquestion.html (hunk 2 above)
Comment (aishwarya-bm) [reply screens/lastquestion.html:52]: Oh yep

**Pull request merged**